pkg/api/v1beta2/types.go

```diff
@@ -653,6 +672,8 @@ const (
 	ResourceCPU ResourceName = "cpu"
 	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
 	ResourceMemory ResourceName = "memory"
+	// CPU Frequency, in Khz
+	ResourceCPUFrequency ResourceName = "cpu_frequency_khz"
```

Comment:
This cannot be a resource. Resources are shared pools of fungible units from which the scheduler allocates.